tests/Support/Data/dump.sql

```diff
@@ -244,7 +244,9 @@ INSERT INTO `wp_options` (`option_id`, `option_name`, `option_value`, `autoload`
 (127, 'widget_wishlistwidget',  'a:1:{s:12:\"_multiwidget\";i:1;}', 'yes'),
 (128, 'WishListMemberOptions_MigrateLevelData', '1',  'yes'),
 (129, 'WishListMemberOptions_MigrateContentLevelData',  '1',  'yes'),
-(130, 'woocommerce_version',  '9.7.1',  'yes');
+(130, 'woocommerce_version',  '10.3.5',  'yes'),
+(131, 'woocommerce_coming_soon', 'no',  'yes'),
```

Comment:
This is possibly overkill, but it ensures the store is live and redirects to cart don't happen, to ensure the same environment for running tests